extensions/wikia/PageShare/PageShareHelper.class.php

```diff
@@ -17,6 +17,25 @@ public static function getIcon( $service ) {
 		return $icon;
 	}
 
+	/**
+	 * Get language for Page Share service.
+	 * For Anon users use the browser language.
+	 * For logged in user use user's language.
+	 * Both values can be overwritten by ?uselang parameter.
+	 *
+	 * @return String language
+	 */
+	public static function getLangForPageShare() {
+		global $wgLang, $wgRequest, $wgUser;
+
+		if ( $wgUser->isAnon() ) {
+			$lang = substr($_SERVER['HTTP_ACCEPT_LANGUAGE'], 0, 2);
```

Comment:
Can we use http://php.net/http_negotiate_language? Do we have a list of supported languages?
Here is some example http://stackoverflow.com/questions/6038236/http-accept-language